PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,66 @@
+# Model Name
+
+## Use-cases
+Model use cases
+
+## Description
+Description of model
+
+## Model
+
+ |Model        |Download  | Download (with sample test data)|ONNX version|Accuracy |
+|-------------|:--------------|:--------------|:--------------|:--------------|
+|Model Name       |ONNX Model download link with size  |  tar file containing ONNX model and synthetic test data|ONNX version used for conversion|Accuracy values |
+<hr>
+
+## Inference
+Step by step instructions on how to use the pretrained model and link to an example notebook/code. This section should ideally contain:
+
+### Input to model
+Input to network (Example: 224x224 pixels in RGB)
+
+### Preprocessing steps
+Preprocessing required
+
+### Output of model
+Output of network
+
+### Postprocessing steps
+Post processing and meaning of output
+<hr>
+
+## Dataset (Train and validation)
+This section should discuss datasets and any preparation steps if required.
+<hr>
+
+## Validation accuracy
+Details of experiments leading to accuracy and comparison with the reference paper.
+<hr>
+
+## Training
+Training details (preprocessing, hyperparameters, resources and environment) along with link to a training notebook (optional)
+<hr>
+
+## Validation
+Validation script/notebook used to obtain accuracy reported above along with details of how to use it and reproduce accuracy.
+<hr>
+
+## References
+Link to references
+<hr>
+
+## Contributors
+Contributors' name
+<hr>
+
+## Acknowledgements
+Thank anyone whose existing work was used
+<hr>
+
+## Keywords
```

Comment:
Let's remove Keywords. The full text will be indexed for search purposes anyways and the keywords don't add value, only add opportunity for mischief.

Reply:
removed keywords from all model readme pages